synapse/logging/context.py

```diff
@@ -460,10 +478,10 @@ class LoggingContextFilter(logging.Filter):
             missing fields
     """
 
-    def __init__(self, **defaults):
+    def __init__(self, **defaults) -> None:
         self.defaults = defaults
 
-    def filter(self, record):
+    def filter(self, record) -> bool:  # FIXME: Use Literal[True] ?
```

Comment:
`Literal[True]` works.

Reply:
I've not made this change yet, as only python 3.8 has `Literal` present, unless the project is willing to add a dependency upon `typing_extensions`.

Reply:
fwiw, we already have a dependency on typing-extensions (currently on v3.7.4)